inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/enums/Constants.java

```diff
@@ -69,4 +80,27 @@ public class Constants {
 
     public static final String SEPARATOR = ":";
 
+    /**
+     * Generate the Job name through {@link ProcessForm}: <br/> 
+     * When the ProcessForm is {@link GroupResourceProcessForm}, the format of the job name is 'InLong-Sort-{Group ID}-{Stream ID}', 
+     * otherwise take the  {@link Constants#DEFAULT_SORT_JOB_NAME}: 'InLong-Sort-Job'. 
+     */
+    public static Function<ProcessForm, String> SORT_JOB_NAME_GENERATOR = (ProcessForm processForm) -> {
+        if (processForm instanceof GroupResourceProcessForm) {
+            GroupResourceProcessForm groupResourceProcessForm = (GroupResourceProcessForm) processForm;
+            String groupId = groupResourceProcessForm.getInlongGroupId();
+            List<InlongStreamInfo> streamInfos = groupResourceProcessForm.getStreamInfos();
+            if (streamInfos == null || streamInfos.isEmpty()) {
+                return String.format(Constants.SORT_JOB_NAME_TEMPLATE, groupId, "NO_STREAM");
+            }
+            String streamIds =
```

Comment:
Will there be a lot of InlongStreams that cause the job name to be too long?

Reply:
In our design, one InlongGroup corresponds to one Flink job, maybe just using the InlongGroupId in the job name is enough?